### DISCUSSION TITLE
页面样式缺失

### DISCUSSION THREAD

**Discussion by heronhe** [umijs/umi#11315] [Q&A]
登录成功进入页面，部分样式缺失，刷新页面之后就可以

第一次进入样式
![WX20230626-113940@2x](https://github.com/umijs/umi/assets/27722383/c1073c4e-f0a5-4f74-9649-87dec54beac4)
刷新页面之后的样式
![WX20230626-113953@2x](https://github.com/umijs/umi/assets/27722383/6ffc7ccb-3093-4a71-9e90-e1f0f706ee87)

**Comment by fz6m** [comment]
需要给一个最小复现。
